Poker.BE/Poker.BE.Domain.Tests/Security/UserManagerTests.cs

```diff
@@ -8,88 +8,128 @@
 
 namespace Poker.BE.Domain.Security.Tests
 {
-    [TestClass()]
-    public class UserManagerTests
-    {
+	[TestClass()]
+	public class UserManagerTests
+	{
 
-        [TestMethod()]
-        public void AddUserTest()
-        {
-            //Arrange
-            var userManager = new UserManager();
+		[TestMethod()]
+		public void AddUserTest()
+		{
+			//Arrange
+			var userManager = new UserManager();
 
-            //Act
-            var res1 = userManager.AddUser("yossi", "password", 100);
-            var res2 = userManager.AddUser("yossi", "mypassword", 200);
-            userManager.RemoveUser("yossi");
-            var res3 = userManager.AddUser("yossi", "mypassword", 200);
-            var res4 = userManager.AddUser("dana", "pass", 100);
-            var res5 = userManager.AddUser("bill", "password", -100);
+			//Act
+			var res1 = userManager.AddUser("yossi", "password", 100);
+			Exception res2 = null;
+			try
+			{
+				userManager.AddUser("yossi", "mypassword", 200);
+			}
+			catch (Exception ex)
+			{
+				res2 = ex;
+			}
+			userManager.RemoveUser("yossi");
+			var res3 = userManager.AddUser("yossi", "mypassword", 200);
+			Exception res4 = null;
+			try
+			{
+				userManager.AddUser("dana", "pass", 100);
+			}
+			catch (Exception ex)
+			{
+				res4 = ex;
+			}
+			Exception res5 = null;
+			try
+			{
+				userManager.AddUser("bill", "password", -100);
+			}
+			catch (Exception ex)
+			{
+				res5 = ex;
+			}
 
-            //Assert
-            Assert.IsTrue(res1);
-            Assert.IsFalse(res2);
-            Assert.IsTrue(res3);
-            Assert.IsFalse(res4);
-            Assert.IsFalse(res5);
-        }
+			//Assert
+			Assert.IsTrue(userManager.CheckExistingUser(res1.UserName));
+			Assert.IsNotNull(res2);
+			Assert.IsTrue(userManager.CheckExistingUser(res3.UserName));
+			Assert.IsNotNull(res4);
+			Assert.IsNotNull(res5);
+		}
```

Comment:
## Tip:
try to avoid log and massive tests. use small ones → you will find it much easier to write several small tests then one massive.
![image](https://cloud.githubusercontent.com/assets/9889268/26473973/149594d0-41b7-11e7-84b2-b532dbc8578e.png)